app/assets/javascripts/discourse/templates/preferences/account.hbs

```diff
@@ -161,6 +161,74 @@
   </div>
 {{/if}}
 
+{{#if canCheckEmails}}
```

Comment:
@nbianca I think we need to add a simple QUnit acceptance test here for this feature.